.env.development

```diff
@@ -9,3 +9,6 @@ DQT_CLIENT_HOST=https://s118d01funcappfapp.azurewebsites.net/
 
 ADMIN_ALLOWED_IPS=::1,127.0.0.1
 ENVIRONMENT_NAME=local
+
+# google cloud storage
+STORAGE_CREDENTIALS=./google_cloud_storage.json
```

Comment:
@d-a-v-e wonder if this should be optional on DEV similar to how it is for the old GA as per the README.md

Reply:
moved to README